### ISSUE TITLE
Can't run on raspbian due to sqlite

### ISSUE THREAD

**Issue by andig** [pimatic/pimatic#1001]
Probably off topic, but after following the install guide I'm unable to run:

```
pi@raspberrypi:~/pimatic-app $ sudo node_modules/pimatic/pimatic.js
22:42:21.793 2016-10-27 Thursday
22:42:29.163 [pimatic] Starting pimatic version 0.9.34
Knex: run
$ npm install sqlite3 --save
Error: Cannot find module '/home/pi/pimatic-app/node_modules/sqlite3/lib/binding/node-v46-linux-arm/node_sqlite3.node'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/home/pi/pimatic-app/node_modules/sqlite3/lib/sqlite3.js:4:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (/home/pi/pimatic-app/node_modules/pimatic/node_modules/coffee-script/lib/coffee-script/register.js:45:36)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Client._driver (../../../src/dialects/sqlite3/index.js:37:12)
    at Client.initializeDriver (../src/client.js:153:26)
    at Client (../src/client.js:36:10)
    at Client_SQLite3 (../../../src/dialects/sqlite3/index.js:19:10)
    at new Client (../../src/util/make-client.js:15:18)
    at Knex (../src/index.js:37:19)
    at /home/pi/pimatic-app/node_modules/pimatic/lib/database.coffee:71:17
    at tryCatcher (/home/pi/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/pi/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/release/promise.js:503:31)
    at Promise._settlePromise (/home/pi/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/release/promise.js:560:18)
    at Promise._settlePromiseCtx (/home/pi/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/release/promise.js:597:10)
    at Async._drainQueue (/home/pi/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/release/async.js:131:12)
    at Async._drainQueues (/home/pi/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/release/async.js:136:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/pi/pimatic-app/node_modules/pimatic/node_modules/bluebird/js/release/async.js:16:14)
```

Fixed by manually installing `sqlite3`. Is there anything we could do about this?

**Comment by rubenoost** [comment]
I have also ran into some issues when running on a Raspberry Pi B1, which one do you use? Following the guide on a Raspberry Pi 3 works perfectly.

**Comment by mwittig** [comment]
I'll prepare a beta release with the current sqlite3 version for testing

**Comment by mwittig** [comment]
should be fixed with pimatic@0.9.37
